indico/modules/events/registration/templates/management/regform_reglist.html

```diff
@@ -150,19 +150,19 @@
                         <ul class="i-dropdown">
                             <li>
                                 <a href="#" class="icon-checkmark js-requires-selected-row disabled js-modify-status"
-                                   data-href="{{ url_for('.registrations_modify_status', regform) }}"
-                                   data-flag="1"
+                                   data-href="{{ url_for('.registrations_approve', regform) }}"
                                    data-method="POST"
                                    data-confirm="{% trans %}Do you really want to approve the selected registrations? This will trigger a notification email for each registrant.{% endtrans %}">
                                     {%- trans %}Approve registrations{% endtrans -%}
                                 </a>
                             </li>
                             <li>
-                                <a href="#" class="icon-close js-requires-selected-row disabled js-modify-status"
-                                   data-href="{{ url_for('.registrations_modify_status', regform) }}"
-                                   data-flag="0"
-                                   data-method="POST"
```

Comment:
we want to keep POST here - see my comment above about `request.form`

(selecting many registrations without POST would result in a VERY long query string otherwise)